netflix/src/redux/films/films.ts

```diff
@@ -0,0 +1,47 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { GENRES } from "../../interfaces/enums/lists.enums";
+
+export const filmSlice = createSlice({
+  name: "film",
+  initialState: {
+    films: [],
+    currentFilter: {
+      currentGenre: GENRES.ALL,
+      currentSort: "",
+      currentSearch: "",
+    },
+    results: undefined,
```

Comment:
Technically, this could break your `singleSourceOfTruth` requirement.
Number of results should always be `films.length` (unless you have pagination).